src/sentry/mail/notifications.py

```diff
@@ -19,7 +19,7 @@
 
 def get_headers(notification: BaseNotification) -> Mapping[str, Any]:
     headers = {
-        "X-SMTPAPI": json.dumps({"category": notification.get_category()}),
+        "X-SMTPAPI": json.dumps({"category": notification.category}),
```

Comment:
@mgaeta You're going to need to propagate this change (and perhaps others) in the getsentry notifications and merge the PRs together.